runtime/thunk/thunk.ts

```diff
@@ -146,6 +147,24 @@ async function doFindAndExecutePackFunction<T extends FormulaSpecification>({
       );
       return response;
     }
+    case FormulaType.ValidateParameters: {
```

Comment:
Does implementation actually require a new top-level FormulaType for this? I was hoping we could use FormulaType.Metadata and just have a new MetadataFormulaType for this, but I don't know if that creates issues if the interface diverges from other metadata formulas.